subworkflows/nf-core/fasta_binning_concoct/meta.yml

```diff
@@ -0,0 +1,75 @@
+name: "fasta_binning_concoct"
+description: Runs the CONCOCT workflow of contig binning
+keywords:
+  - concoct
+  - binning
+  - metagenomics
+  - contigs
+modules:
+  - concoct/cutupfasta
+  - concoct/concoctcoveragetable
+  - concoct/concoct
+  - concoct/mergecutupclustering
+  - concoct/extractfastabins
+input:
+  - ch_fasta:
+      type: file
+      description: |
+        Structure: [ val(meta), path(fasta)]
```

Comment:
Ups, I understood uncorrectly one of your comments in your review of the chromap subworkflow and thought that structure was indeed a key in the yml 🤦

Reply:
Yes, sorry :sweat: But maybe it _should_ be a key... but I think this is still under discussion.